### DISCUSSION TITLE
Specimen symmetry for compressed samples

### DISCUSSION THREAD

**Discussion by Samarendra0101** [mtex-toolbox/mtex#2158] [Ask Anything]
I have done uniaxial compression on an Aluminum sample and have done x-ray texture measurement on the compressed sample. Can anyone kindly tell me what "specimen symmetry" should I use to analyze the pole figure data? Because I believe orthorhombic symmetry ('mmm') should not work for a compressed sample.
Thanks in advance.

**Comment by ralfHielscher** [comment]
Best is always to work without specimen symmetry.
Ralf.
